plugins/welcome/src/components/WelcomePage/WelcomePage.tsx

```diff
@@ -38,7 +38,7 @@ import {
   configApiRef,
 } from '@backstage/core';
 
-const WelcomePage: FC<{}> = () => {
+const WelcomePage = ({}) => {
```

Comment:
here you can just do `() =>` without the `{}`

Reply:
Thanks! Will do.